languages/php.rst

```diff
@@ -33,6 +33,9 @@ PHP extensions can be enabled/disabled under the ``runtime.extensions`` and
         disabled_extensions:
             - sqlite3
 
+To install specific versions or build an extension not available on PECL or on
+SymfonyCloud, use the :ref:`php-ext-install <php-ext-install>` script instead.
```

Comment:
The cross-reference should also be added in "Custom PHP Extensions" (it was missing)

Reply:
Done